base/notification/notification.go

```diff
@@ -80,5 +80,6 @@ func MakeNotification(inventoryID, accountName, eventType string, events []Event
 		AccountID: accountName,
```

Comment:
is it expected to send both `AccountID` and `OrgID`?

Reply:
`notifications-backend` barely just started reading `OrgID` from events, so I suppose they are still working on it, but sending it should not break anything now. Maybe `AccountID` will become deprecated later on, you actually should know better if this is direction we are going to, since you've tackled `OrgID` :grin:
I do not think it is safe to remove `AccountID` now.